packages/web-react/src/components/Dropdown/README.md

```diff
@@ -39,7 +40,94 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
+| `UNSAFE_className` | `string`                  | —        | ✕        | DropdownTrigger custom classname |
+| `UNSAFE_style`     | `CSSProperties`           | —        | ✕        | DropdownTrigger custom style     |
+
+### DropdownModernPopover
+
+| Name               | Type                      | Default | Required | Description                      |
+| ------------------ | ------------------------- | ------- | -------- | -------------------------------- |
+| `children`         | [`string` \| `ReactNode`] | —       | ✔        | Content of trigger element       |
+| `UNSAFE_className` | `string`                  | —       | ✕        | DropdownContent custom classname |
+| `UNSAFE_style`     | `CSSProperties`           | —       | ✕        | DropdownContent custom style     |
+
+### UncontrolledDropdown
+
+| Name               | Type                                             | Default       | Required | Description                                    |
+| ------------------ | ------------------------------------------------ | ------------- | -------- | ---------------------------------------------- |
+| `enableAutoClose`  | `bool`                                           | `true`        | ✕        | Enables close on click outside of Dropdown     |
+| `fullWidthMode`    | [`DropdownFullwidthMode`][dropdownfullwidthmode] | `off`         | ✕        | Full-width mode                                |
+| `id`               | `string`                                         | `<random>`    | ✕        | Component id                                   |
+| `onAutoClose`      | `(event: Event) => void`                         | —             | ✕        | Callback on close on click outside of Dropdown |
+| `placement`        | [Placement dictionary][dictionary-placement]     | `bottom-left` | ✕        | Alignment of the component                     |
+| `UNSAFE_className` | `string`                                         | —             | ✕        | Wrapper custom classname                       |
+| `UNSAFE_style`     | `CSSProperties`                                  | —             | ✕        | Wrapper custom style                           |
+
 [dropdown]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/src/scss/components/Dropdown
+[DropdownModern]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/src/scss/components/DropdownModern
 [dropdownbreakpoint]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/types/dropdown.ts#L11
 [dropdownfullwidthmode]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/types/dropdown.ts#L19
 [dictionary-placement]: https://github.com/lmc-eu/spirit-design-system/tree/main/docs/DICTIONARIES.md#placement
+[Deprecated]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-react/README.md#deprecations
```

Comment:
i think we write all links in lowercase, so please can you fix it?

Reply:
fixed